### PR TITLE
Other examples added

### PR DESCRIPTION
**Pull request by gael-millot**
The end of the page points to https://github.com/tidyverse/ggplot2/issues/3839 to describe special behaviors of group

### CODE REVIEW

File: R/aes-group-order.r [hunk 1]
```diff
@@ -72,5 +72,6 @@
 #' # we again need to override the default grouping for that layer with aes(group = Subject)
 #' boysbox <- boysbox + geom_boxplot()
 #' boysbox + geom_line(aes(group = Subject), colour = "blue")
-#' }
```
Comment (yutannihilation) [inline R/aes-group-order.r:75]: You cannot delete this `}`.

**Comment by yutannihilation** [comment]
Ah, sorry, please ignore my comment above. It seems I need to run `/ document` after merging this into master...

By the way, what I meant on https://github.com/tidyverse/ggplot2/issues/3839#issuecomment-595291390 was to add some example *code*, not a link to the discussion. I think this PR is better than nothing, but this might not very handy for users.

**Comment by thomasp85** [comment]
@gael-millot are you interested in fixing this up by summarising the points in #3839 rather than linking to it?

**Comment by gael-millot** [comment]
Hi @thomasp85. I sent a new pull request. Thanks.

**Pull request closed without merging**